tools/get-alpine-pkg-source.sh

```diff
@@ -76,7 +76,9 @@ get_ocpairs() {
     awk -F: '
         /^o:/ { origin=$2 }
         /^L:/ { license=$2 }
-        /^c:/ { commit=$2; if (commit == "") { commit = "unknown" }; print origin, commit, license; origin="XXX"; license="" }' < "$1" | sort -u > "$2"
+        /^V:/ { version=$2 }
+        /^c:/ { commit=$2; if (commit == "") { commit = "unknown" }; print origin, version, commit, license; origin="XXX"; license="" }' < "$1" | sort -u > "$2"
```

Comment:
Shouldn't you add version="" here to avoid carrying the version from the previous package?

Reply:
will add the `version=""`

Reply:
added the version. Fixed it